src/Components/Publishing/Header/BasicHeader.tsx

```diff
@@ -148,27 +149,24 @@ const Container = styled.div`
     width: 100%;
 
     @media screen and (min-width: 1250px) {
-      height: ${1100 * VIDEO_RATIO};
+      height: ${1100 * VIDEO_RATIO}px;
     }
-
     ${breakpoint.xl`
       height: ${1100 * VIDEO_RATIO}px;
     `}
-
     ${breakpoint.lg`
       height: ${950 * VIDEO_RATIO}px;
     `}
-
     ${breakpoint.md`
       height: ${800 * VIDEO_RATIO}px;
     `}
-
     ${breakpoint.sm`
       height: ${620 * VIDEO_RATIO}px;
     `}
-
     ${breakpoint.xs`
-      height: ${520 * VIDEO_RATIO}px;
+      height: ${340 * VIDEO_RATIO}px;
```

Comment:
Now that the 🔥  is simmered down, can you remind me again why we made these breakpoints have fixed values? Maybe it's worth a quick SH but unless these values were specific to the design, I'd ideally like to replace these breakpoints with -- `height: calc((100vw - 100) * ${VIDEO_RATIO})` or some dynamic variation of that is possible.

Reply:
These values were computed by Owen and represent the width * a fraction, which returns the proper aspect-ratio-height per the designs. I can try the `vh` approach but note that these values now fit the spec nicely!